.gitignore

```diff
@@ -43,3 +43,9 @@ trace.db/
 *server.config
 *.mov
 *.pb
+lib/
+windows/
+*.dll
+*.csv
+CSVSource/
```

Comment:
when i worked on both pool simultaneously, i had issues with that .*.dll will be fixed when we work on HikVision . this would be able to complete cleaned after the third one ,

Reply:
also .jar -- can't include those either.

Reply:
there is no Jar , dll was removed , done